src/pages/Profile.tsx

```diff
@@ -1,5 +1,222 @@
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getTweetCount",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setTweetCount(result.data);
+    };
+
+    const fetchBio = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getBio",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setBio(result.data);
+    };
+
+    const fetchJoinDate = async () => {
+      const result = await axios.get(
+        "http://localhost:3001/api/profile/getJoinDate",
+        {
+          params: {
+            userId: user.userId,
+          },
+        }
+      );
+      setJoinDate(result.data);
+    };
+    fetchBio();
+    fetchTweetCount();
+    fetchJoinDate();
+  }, [value, user]);
+  return (
+    <>
+      <Box width={650}>
+        <div style={styles.rightBound}>
+          {/**Not sure about this */}
+          <div style={styles.header}>
+            {" "}
+            <Typography sx={styles.arrowText}>{user.username}</Typography>
+            <button style={styles.arrowBtn}>
+              <ArrowBackIcon></ArrowBackIcon>
+            </button>
+            <Typography sx={styles.tweetCountStyle}>
+              {tweetCount} Tweets
+            </Typography>
+          </div>
+          <div style={{ position: "relative" }}>
+            <img
+              src={process.env.PUBLIC_URL + "/blue.jpg"}
+              alt="temp"
+              width="650px"
+              height="200px"
+            ></img>
+            <Avatar
+              alt="profile picture"
+              src={process.env.PUBLIC_URL + "/rock.jpg"}
+              sx={styles.avatar}
+            />
+          </div>
+          <Button
+            variant="contained"
+            startIcon={<EditIcon />}
+            sx={styles.editBtn}
+          >
+            Edit Profile
+          </Button>
+          <div style={{ paddingLeft: 15, overflowWrap: "break-word" }}>
+            <Typography variant={"h1"} style={styles.displayName}>
+              {user.displayName}
+            </Typography>
+            <Typography variant={"h2"} style={styles.usernameDisplay}>
+              @{user.username}
+            </Typography>
+            <Typography style={styles.bioDisplay}>{bio}</Typography>
+            <div style={{ alignItems: "center", display: "flex" }}>
+              <CalendarMonthIcon></CalendarMonthIcon>
+              <Typography>Joined {joinDate}</Typography>
+            </div>
+            <Button sx={styles.followercountbtn}>
+              <div>
+                <b>500M</b> Followers
+              </div>
+            </Button>
+            <Button sx={styles.followingcountbtn}>
+              <div>
+                <b>0</b> Following
+              </div>
+            </Button>
+          </div>
+          <Tabs value={value} onChange={handleChange}>
```

Comment:
Good use of Tabs component!

Reply:
One nitpick, I think since `handleChange` isn't being reused and its logic isn't overly complex, we can inline it as an arrow function. My suggestion is to do something like this instead:

```suggestion
          <Tabs value={value} onChange={(_: React.SyntheticEvent, newValue: string) => setValue(newValue)}>
```

I prefer inlining in this case because it's not a lot of code. If the onChange was a lot more code I'd agree with this approach to not bloat the jsx